frontends/pytorch/csrc/builder/graph_importer.h

```diff
@@ -0,0 +1,117 @@
+//===- graph_importer.h -----------------------------------------*- C++ -*-===//
+//
+// This file is licensed under a pytorch-style license
+// See frontends/pytorch/LICENSE for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef NPCOMP_FRONTENDS_PYTORCH_CSRC_GRAPH_IMPORTER_H
+#define NPCOMP_FRONTENDS_PYTORCH_CSRC_GRAPH_IMPORTER_H
+
+#include <memory>
+
+#include "../pybind.h"
+#include "func_builder.h"
+
+#include "mlir-c/IR.h"
+
+#include <torch/csrc/jit/api/compilation_unit.h>
+#include <torch/csrc/jit/ir/ir.h>
+
+namespace torch_mlir {
+
+/// Main entry-point for importing graphs.
+/// In torch terminology, a Graph is a function. We materialize it as at
```

Comment:
"We materialize it" reads to me like GraphImporter will materialize it. Maybe "later in the compiler, we might specialize multiple versions of it".